app/app.go

```diff
@@ -768,9 +768,17 @@ func (app *App) AutoCliOpts() autocli.AppOptions {
 		}
 	}
 
+	moduleOptions := runtimeservices.ExtractAutoCLIOptions(app.ModuleManager.Modules)
+
+	// skip the comet consensus module autocli commands for v4.0.0
+	// to be re-added in v4.1.0 when the system is full on v0.50
```

Comment:
I don't understand why it is safe to re-add this in v4.1.0.

The multiplexer needs to be able to handle block syncing historical blocks from genesis. If we re-add this in v4.1.0 what happens when a v4.1.0 multiplexer block syncs through v3.x blocks and a user queries:

```
$ celestia-appd query consensus comet block-latest
```

Reply:
The query works, this is just the query commands that has an issue because of AutoCLI.
You can query via grpc or grpc gateway, as usual.

Reply:
Why is it broken on v4.x.x and why is it safe to add back in v4.1.0?

Reply:
agree, some clarity here would be helpful :slightly_smiling_face: . I'm missing how this fixes the issue permanently. 

is the issue peristent if both the client and server are running v4 or does it occur then as well? 

if this is a permanent problem, assuming the feature is not important (querying the block from the cli seems like it falls in that category), then I'd be okay returning an error for v3 blocks but ofc would prefer fixing the root issue